cmake/modules/FindUFE.cmake

```diff
@@ -46,6 +46,10 @@ if(UFE_INCLUDE_DIR AND EXISTS "${UFE_INCLUDE_DIR}/ufe/ufe.h")
 
     if("${UFE_MAJOR_VERSION}" STREQUAL "0")
         math(EXPR UFE_PREVIEW_VERSION_NUM "${UFE_MINOR_VERSION} * 1000 + ${UFE_PATCH_LEVEL}")
+    elseif("${UFE_VERSION}" STREQUAL "3.0.0")
+        # Temporary. Once next Maya PR is released with UFE v3.0.0 this should
+        # be removed (along with all the UFE_PREVIEW_VERSION_NUM checks).
+        set(UFE_PREVIEW_VERSION_NUM 3014)
```

Comment:
This is what I did when we transitioned to Ufe v2. I commit this first and then update Maya to Ufe v3. Then after a Maya PR ships with Ufe v3 we can fixup the Ufe preview checks and remove this.